### ISSUE TITLE
NTR: alpha-oxoadipic acid high in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8518]
New term request alpha-oxoadipic acid high in urine (segal_030505171524): 2-oxoadipic aciduria  UMLS: 4011938

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000266
- label: 2-oxoadipic aciduria
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: PMID:23141293
- def: The amount of 2-oxoadipic acid in the urine, normaliyed for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8518
- synonyms: Alpha-oxoadipic acid high in urine(NONE)
